virtool/groups/oas.py

```diff
@@ -79,8 +79,14 @@ class EditGroupSchema(BaseModel):
         default={},
     )
 
+    name: Optional[constr(min_length=1)] = Field(
+        description="a name for the group", default=None
```

Comment:
Remove `default` value. If a name is not provided, it will be excluded from the update using `exclude_unset`.